libraries/cmake/source/modules/utils.cmake

```diff
@@ -144,17 +153,6 @@ function(importSourceSubmodule)
         "${directory_path}/${submodule_to_patch}"
         "${patched_source_dir}"
       )
-
-      if(patchSubmoduleSourceCode_Patched)
-        list(FIND ARGS_SHALLOW_SUBMODULES "${submodule_to_patch}" shallow_clone)
-        if(${shallow_clone} EQUAL -1)
-          set(shallow_clone false)
-        else()
-          set(shallow_clone true)
-        endif()
-
-        initializeGitSubmodule("${directory_path}/${submodule_to_patch}" ${ARGS_NO_RECURSIVE} ${shallow_clone})
```

Comment:
Is this intentionally removed?

Reply:
Yes, that was the previous way of "resetting" the submodule in the source directory, since the folder was completely moved out.